doc/qbk/02_9_conversion.qbk

```diff
@@ -13,101 +13,303 @@
 
 While the __value__ container makes it easy to create ad-hoc structures,
 often it is necessary to convert between JSON and specific user-defined
-types. The library provides a set of customization points to enable
-conversions between objects of type `T` and a JSON value by:
+types. Conversions from a type `T` to __value__ are done by __value_from__.
+Converting in the opposite direction is done with __value_to__.
```

Comment:
This avoids the switch from singular to plural:

"A conversions from a type `T` to __value__ is done by __value_from__. The conversion in the opposite direction is done with __value_to__."